build.gradle

```diff
@@ -222,3 +228,11 @@ if (System.env.TRAVIS == 'true') {
     }
   }
 }
+
+// Temporary work around to get compilation working during development
+// Considering moving to a separate module
+// https://discuss.gradle.org/t/kotlin-groovy-and-java-compilation/14903/10
+compileGroovy.dependsOn = compileGroovy.taskDependencies.mutableValues - 'compileJava'
```

Comment:
This is really ugly so Im hoping to move it to a seperate module. Im just not sure where it should live in the project.